addons/binding/org.openhab.binding.hyperion/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,35 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Hyperion Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.hyperion;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Export-Package: 
+ org.openhab.binding.hyperion,
+ org.openhab.binding.hyperion.handler
+Import-Package: 
+ com.google.common.collect,
```

Comment:
We try to get rid of guava. Please remove this dependency and replace it by JDK API. See https://www.eclipse.org/smarthome/documentation/development/guidelines.html#c-language-levels-and-libraries for reference.

Reply:
done